src/game/WorldHandlers/LFGMgr.cpp

```diff
@@ -1177,6 +1177,9 @@ void LFGMgr::SendQueueStatus()
                         case PLAYER_ROLE_DAMAGE:
                             playerWaitTime = m_dpsWaitTime[dungeonId].time;
                             break;
+                        default:
+                            playerWaitTime = 0; //is this the right default?
```

Comment:
If I am guessing currently, this line should be like this below to include the map for the player wait time.
```cpp
playerWaitTime = m_avgWaitTime[dungeonId].time;
```

Reply:
Perfect!